.bazelrc

```diff
@@ -2,11 +2,122 @@
 # running as daemons, and cache SourceFile AST's to reduce parse time.
 build --strategy=TypeScriptCompile=worker
 
-# Performance: avoid stat'ing input files
-build --watchfs
+# Enable debugging tests with --config=debug
+test:debug --test_arg=--node_options=--inspect-brk --test_output=streamed --test_strategy=exclusive --test_timeout=9999 --nocache_test_results
 
+###############################
+# Filesystem interactions     #
+###############################
+
+# Create symlinks in the project:
+# - dist/bin for outputs
+# - dist/testlogs, dist/genfiles
+# - bazel-out
+# NB: bazel-out should be excluded from the editor configuration.
+# The checked-in /.vscode/settings.json does this for VSCode.
+# Other editors may require manual config to ignore this directory.
+# In the past, we say a problem where VSCode traversed a massive tree, opening file handles and
+# eventually a surprising failure with auto-discovery of the C++ toolchain in
+# MacOS High Sierra.
+# See https://github.com/bazelbuild/bazel/issues/4603
+build --symlink_prefix=dist/
+
+# Disable watchfs as it causes tests to be flaky on Windows
+# https://github.com/angular/angular/issues/29541
+build --nowatchfs
```

Comment:
Perhaps we should have a platform specific set of rc files for this type of situation?  Since we would like `--watchfs` to be on for unix based systems, but need it disabled for windows.

Reply:
If there is a way to make them be used automatically based on the platform outside of CI, that would be a good idea.

Reply:
I don't know that there is a way to automatically choose the platform.

I wasn't thinking about how this wasn't CI specific, we can just leave it as `--nowatchfs`.  One day maybe we will be able to do a platform based configurations.